src/main/java/org/akhq/repositories/RecordRepository.java

```diff
@@ -76,6 +77,9 @@
     @Value("${akhq.clients-defaults.consumer.properties.max.poll.records:50}")
     protected int maxPollRecords;
 
+    @Value("${akhq.kafka-max-message-length}")
```

Comment:
use `akhq.topic-data.kafka-max-message-length` and please add some docs [here](https://github.com/tchiotludo/akhq/blob/dev/docs/docs/configuration/akhq.md) and add the value [here](https://github.com/tchiotludo/akhq/blob/dev/application.example.yml)